src/styles/catalog.scss

```diff
@@ -0,0 +1,16 @@
+.catalog {
+  margin: 0 auto;
```

Comment:
Don't set margin for BEM block. Add additional class to make them element of other block

Reply:
added: conteiner__catalog

Reply:
I didn't see changes